appendix/teachmeqiskit2018/README.md

```diff
@@ -0,0 +1,18 @@
+## Teach Me QISKit Award 2018: Top Submissions
+
+This folder contains the top submissions of the Teach Me QISKit Award 2018.
+Most of the tutorials were written and tested with qiskit-sdk-py version 0.4.15.
+
+Here is the list of the top submissions (in alphabetical order).
+
+- [CHSH Game](chsh_game/CHSH%20game-tutorial.ipynb) - play a quantum game.
```

Comment:
could we give [name](link) by Awesome Quantum - some quick description

Reply:
The link you gave is broken and I cannot find it. I modified the name and its description like below:
Awesome Quantum - get used to the quantum weirdness by playing a quantum game.

Reply:
Done?

Reply:
Done.